ci/run-benchmark.yaml

```diff
@@ -2,7 +2,7 @@ platform: linux
 image_resource:
   type: registry-image
   source:
-    repository: ((image_registry))/eq-app-k8s-deploy-image
```

Comment:
This isn't directly mentioned on the card, only `eq-app-deploy-image` so I need to double check.

Reply:
Yeah, it's a bit odd as I couldn't spot which of our repos were creating the image named `eq-app-k8s-deploy-image` 🤔